### ISSUE TITLE
`maid watch` might be a friendlier synonym to `maid daemon`

### ISSUE THREAD

**Issue by brandondrew** [benjaminoakes/maid#141]
`maid daemon` makes sense to programmers and Unix hackers, but doesn't really fit with the metaphor of a maid doing your cleaning for you, and doesn't make sense to people with less familiarity with what's going on behind the scenes.  It's a good name, but perhaps a friendlier synonym for the masses might be `maid watch`, or perhaps even better, `maid service`.

**Comment by benjaminoakes** [comment]
I'd be for this (pull requests welcome).

**Comment by coaxial** [comment]
Going through old issues and triaging.

Is there still an interest for this feature? Popularity will affect implementation prioritization for a future release.

**Comment by benjaminoakes** [comment]
I agree with this naming. It would be a good first issue for someone.
